.github/workflows/versions.yml

```diff
@@ -0,0 +1,42 @@
+name: versions
+
+on:
+  push:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.7
+      JDK_VERSION: 14
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}    
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set Up JDK ${{ env.JDK_VERSION }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JDK_VERSION }}
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install Pipenv and Dependencies
+        run: |
+          python -m pip install --upgrade pipenv wheel
+      - name: Update OpenSearch Manifests
+        run: |
+          ./bundle-workflow/manifests.sh update
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

Comment:
Could this pile up duplicate pull requests?  Do we have any control over managing duplication?

Reply:
The plugin assumes a branch name (which here I didn't specify so it will just be called "patch") and if it sees a PR from a branch called "patch", it will update it instead of creating a new one. It's pretty smart like that.

Reply:
Awesome! Could we add a branch name that is based off of the version number?  This way if there is a patch release and a minor release updates at the same time separate pull requests are created.

Reply:
Not so easy to know what's being added, would require returning something meaningful from the build that can be parsed by the workflow. I did update the branch name to `manifests/auto` so that the auto-pr doesn't walk over other PRs.

Reply:
Setting `branch:` [didn't work](https://github.com/dblock/opensearch-build/runs/3612091607?check_suite_focus=true) on my fork with a permissions error that doesn't happen with defaults, so I undid that.